std_msgs/Header header
float32 num
float32 blue_num
float32 red_num
float32 death_num
float32 lost_frame_counter
Object[] object
